Traceback (most recent call last):
  File "/home/runner/work/MultiAssayExperiment/MultiAssayExperiment/.tox/docs/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/runner/work/MultiAssayExperiment/MultiAssayExperiment/.tox/docs/lib/python3.9/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/runner/work/MultiAssayExperiment/MultiAssayExperiment/.tox/docs/lib/python3.9/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/hostedtoolcache/Python/3.9.20/x64/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/runner/work/MultiAssayExperiment/MultiAssayExperiment/.tox/docs/lib/python3.9/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/runner/work/MultiAssayExperiment/MultiAssayExperiment/.tox/docs/lib/python3.9/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/runner/work/MultiAssayExperiment/MultiAssayExperiment/.tox/docs/lib/python3.9/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from rich import print as pprint
pprint("row names:", mae.get_row_names())
pprint("column names:", mae.get_column_names())
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[8], line 1[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mrich[39;00m [38;5;28;01mimport[39;00m [38;5;28mprint[39m [38;5;28;01mas[39;00m pprint
[1;32m      2[0m pprint([38;5;124m"[39m[38;5;124mrow names:[39m[38;5;124m"[39m, mae[38;5;241m.[39mget_row_names())
[1;32m      3[0m pprint([38;5;124m"[39m[38;5;124mcolumn names:[39m[38;5;124m"[39m, mae[38;5;241m.[39mget_column_names())

[0;31mModuleNotFoundError[0m: No module named 'rich'

